oh.bat

```diff
@@ -122,11 +122,12 @@ set DATABASE_USER=isf
 set DATABASE_PASSWORD=isf123
 
 set DICOM_MAX_SIZE="4M"
-set DICOM_STORAGE="FileSystemDicomManager" REM SqlDicomManager
+set DICOM_STORAGE="FileSystemDicomManager"
 set DICOM_DIR="data/dicom_storage"
 
-set OH_DIR="."
+set OH_DIR="oh"

```

Comment:
this should be set back to "." like the others?

Reply:
Yes right, corrected.